ros_buildfarm/templates/snippet/builder_system-groovy_verify-upstream.xml.em

```diff
@@ -1,3 +1,6 @@
+@{
+code = TEMPLATE('snippet/check_recursive_upstream_projects.groovy.em')
+}@
```

Comment:
Absolutely. Thanks for reporting this. The line was supposed to assign the content of the file to the variable `code`. But instead it outputs it in-place.

Please see #448 for the fix.